Header header
float32 magnetic_x
float32 magnetic_y
float32 magnetic_z
geometry_msgs/Twist twist
geometry_msgs/Wrench wrench